cute-name-service/.openshiftio/application.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-kind: Template
```

Comment:
Something looks funny here.  I don't think the Template kind is to be removed,  it looks like you are replacing it with the ImageStream

should be similar to this https://github.com/nodeshift-starters/nodejs-rest-http/commit/309e8be180064ad994f8c2ed823fdf5512a467dd#diff-5dce3a731b5aff7080a31a1adcd60ae7

Reply:
yeah I should avoid to edit multiples files at the same time.
I'm going to update that, thanks!